pkg/cli/mirror/mirror.go

```diff
@@ -269,6 +270,41 @@ func (o *MirrorOptions) Run(cmd *cobra.Command, f kcmdutil.Factory) (err error)
 	diskToMirror := len(o.ToMirror) > 0 && len(o.From) > 0
 	mirrorToMirror := len(o.ToMirror) > 0 && len(o.ConfigPath) > 0
 
+	if o.UseOCIFeature {
```

Comment:
I wholeheartedly don't link nested ifs, they read poorly. It's better to invert the condition or if necessary create a function you call, so the overall read of the code is cleaner.